model_compression_toolkit/core/common/constants.py

```diff
@@ -127,4 +127,10 @@
 
 # Jacobian-weights constants
 MIN_JACOBIANS_ITER = 10
-JACOBIANS_COMP_TOLERANCE = 1e-3
\ No newline at end of file
+JACOBIANS_COMP_TOLERANCE = 1e-3
+
+# Quantizers constants (for GPTQ, QAT, etc.)
+FQ_MIN = "min"
```

Comment:
better put these in the trainable_quantizers infrastructure, that all quantizers inherit from